### PR TITLE
Externalizing sidecar image name

### PR DESCRIPTION
**Pull request by freynca**
Externalizing sidecar image name to a configurable property.

### CODE REVIEW

File: event-dispatcher/src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+sk8s.event.dispatcher.sidecarImageName=sk8s/function-sidecar:latest
```
Comment (markfisher) [inline event-dispatcher/src/main/resources/application.properties]: I don't think we want `latest` here as that would pull every time.

**Comment by freynca** [comment]
Changed default to `stable`. The default will be overridden by CI to match the appropriate version of the image.

**Comment by markfisher** [comment]
we need to keep the default as `0.0.1-SNAPSHOT`

File: event-dispatcher/src/main/java/io/sk8s/event/dispatcher/EventDispatcherProperties.java [hunk 2]
```diff
@@ -25,6 +25,7 @@
 public class EventDispatcherProperties {
 
 	private String namespace = "default";
+	private String sidecarImageName = "sk8s/function-sidecar";
```
Comment (markfisher) [inline event-dispatcher/src/main/java/io/sk8s/event/dispatcher/EventDispatcherProperties.java]: we could leave this out if providing via `application.properties` (or if we're going to have a default here, then we don't need `application.properties` anyways, but this should also include the tag)

**Comment by freynca** [comment]
Closing this PR; will re-open a new one

**Pull request closed without merging**